lib/rate-limiter.js

```diff
@@ -12,13 +12,14 @@ class RateLimiter {
    *
    * @param {Object} options
    */
-  constructor (options) {
+  constructor(options) {
     const {
       view,
       max = 60,
       duration = 60 * 1000,
       userAttribute = 'id',
       userLimitAttribute = 'rateLimit',
+      skip = false,
```

Comment:
The `skip` option should be a function. That means, don't provide a default value here